devops/dependencies.json

```diff
@@ -25,15 +25,15 @@
       "root": "{DEPS_ROOT}/tbb/lin"
     },
     "oclcpu": {
-      "github_tag": "2021-WW50",
-      "version": "2021.13.11.0.23",
-      "url": "https://github.com/intel/llvm/releases/download/2021-WW50/oclcpuexp-2021.13.11.0.23_rel.tar.gz",
+      "github_tag": "2022-WW13",
```

Comment:
@alexbatashev, this PR is supposed to update GPU driver, not CPU. There is another PR for updating CPU runtime.